source/layouts/_footer_custom.haml

```diff
@@ -16,7 +16,8 @@
     {:.}
 
     Use RDO
-    : [Quickstart](/install/quickstart)
+    : [PackStack](/install/quickstart)
+    : [QuickStart](/tripleo/)
```

Comment:
Let's be consistent with other changes wrt TripleO and call this "TripleO quickstart". It's also less confusing.